### DISCUSSION TITLE
why is this when I click view page

### DISCUSSION THREAD

**Discussion by Tribagasiskandar** [givanz/Vvveb#30] [General]
![image](https://github.com/givanz/Vvveb/assets/67426865/5db70bb2-1e89-4922-98d7-76424e3b9814)

**Comment by givanz** [comment]
Hi

The file `public/themes/landing/index.html` exists on the server? 

The error is shown on homepage or other page like post etc?

The `view page`  is the button from posts list in admin or other button?

![Screenshot_2023-05-27_13-27-32](https://github.com/givanz/Vvveb/assets/302738/89f97a4d-2602-45d6-b3a9-317a7cb50f3e)
